dev/io.openliberty.microprofile.health.3.0.internal/src/io/openliberty/microprofile/health30/internal/HealthCheckConstants.java

```diff
@@ -33,4 +34,7 @@
     // Default Overall Readiness Status MP Config property name
     public static final String defaultOverallReadinessStatusPropertyName = "mp.health.default.readiness.empty.response";
 
+    // Default Overall Startup Status MP Config property name
+    public static final String defaultOverallStartupStatusPropertyName = "mp.health.default.startup.empty.response";
```

Comment:
Conventionally, constants are all upper case (optionally with underscores).
```suggestion
    public static final String DEFAULT_OVERALL_STARTUP_STATUS_PROPERTY_NAME = "mp.health.default.startup.empty.response";
```
You might want to make that name a bit shorter too?

Reply:
Updated, changed it to `DEFAULT_OVERALL_STARTUP_STATUS`